src/Widgets/DataImageWidget/DataImageWidgetClass.ts

```diff
@@ -0,0 +1,13 @@
+import { provideWidgetClass } from 'scrivito'
+
+export const DataImageWidget = provideWidgetClass('DataImageWidget', {
+  attributes: {
+    alignment: ['enum', { values: ['left', 'center', 'right'] }],
+    alternativeText: 'string',
+    data: 'datalocator',
+    roundCorners: 'boolean',
+    link: 'link',
```

Comment:
* [ ] FYI none of the existing widgets uses the `link` feature. We could drop it to make the widget simpler.

Reply:
I would like to keep it.